test/csv/test_patterns.rb

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# frozen_string_literal: true
+
+require_relative "helper"
+
+class TestCSVPatterns < Test::Unit::TestCase
+  extend DifferentOFS
+
+  def test_pattern_matching_hash
+    case CSV::Row.new(%i{A B C}, [1, 2, 3])
+    in B: b, C: c
```

Comment:
Could you add one more test case for `in B:, **rest`?

Reply:
Updated!